### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by TriForceX**
- Fix rewind time on:
  - Draw timer
  - Scoreboard player time
  - Server idle time
  - Vote time

- Fix number shaders

### CODE REVIEW

**Comment by aufau** [comment]
About scoreplums - I don't necessarily see it as a bug, it was designed this way. Basically copied what I saw in other games and damage plums were visible through walls there. Also in saber combat it's much easier to read them if they are visible through your own player - this is why I ended up making them so huge.

Now I'm not saying they are perfect, I'd love to have a different style of damage plums - higher resolution font, different size/movement pattern etc. Maybe changeable with a cvar - cg_damagePlumsStyle, or just cg_damagePlums 0|1|2|...? But I wouldn't change this exact style, it has been there for a long time and many players are used to it by now.

**Comment by TriForceX** [comment]
The chance to the damage plum beign "hidden" from your own body is very low, the player model and its camera are always moving, also it thats not the case, the damage plum is moving too... ive wanted to add this shader because when you kill someone or you get killed while attacking (with saber or weapon shots), the damage plum can still be seen through walls from your last position.

I know is not something very relevant, is more an improvement than a needed fix.

File: code/game/g_main.c [hunk 1]
```diff
@@ -3090,12 +3090,14 @@ void G_RewindTime( int msec ) {
...
 
-	ADJUST(level.startTime);
 	ADJUST(level.warmupTime);
 	ADJUST(level.intermissiontime);
 	ADJUST(level.roundQueued);
 	ADJUST(level.intermissionQueued);
 	ADJUST(level.exitTime);
+	ADJUST(level.idleTime);
+
+	if(level.startTime < level.time) level.startTime += msec;
```
Comment (aufau) [inline code/game/g_main.c]: I understand why ADJUST(level.startTime) was wrong. But why is `if(level.startTime < level.time)` condition needed?

Comment (TriForceX) [reply code/game/g_main.c]: Ops... you're right, level.time is always equal or higher than startTime, no need for that condition, just the sum of msec

**Comment by TriForceX** [comment]
Ive removed the startTime condition now, and recently noticed voteTime check was missing, so i added it too.

File: code/game/g_main.c [hunk 2]
```diff
@@ -2782,7 +2782,7 @@ void CheckVote( void ) {
 			}
 		}
 	}
-	if ( !level.voteTime ) {
+	if ( !level.voteTime || level.unpauseTime > level.time ) {
```
Comment (aufau) [inline code/game/g_main.c:2785]: Voting during pause was enabled on purpose. It looks like a vote won't be able to pass during pause after this change?

Comment (TriForceX) [reply code/game/g_main.c:2785]: Yeah ... I thought you forgot it, but if that was enabled on purpose (idk why) ignore it... (also the configstring update on rewind)

**Pull request closed without merging**